core/server/common/src/main/java/alluxio/master/journal/noop/NoopPrimarySelector.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.journal.noop;
+
+import alluxio.grpc.NodeState;
+import alluxio.master.PrimarySelector;
+import alluxio.util.interfaces.Scoped;
+
+import java.net.InetSocketAddress;
+import java.util.function.Consumer;
+
+/**
+ * Noop primary selector.
+ */
+public class NoopPrimarySelector implements PrimarySelector {
```

Comment:
do you want to consolidate this with https://github.com/Alluxio/alluxio/blob/master/core/server/master/src/test/java/alluxio/master/AlwaysStandbyPrimarySelector.java?

Reply:
Great idea, I will refactor to use only the existing `AlwaysStandbyPrimarySelector`.